google-sheets/assets/index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Get started with your Twilio Functions!</title>
+    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,300italic,700,700italic">
+    <link rel="stylesheet" href="https://unpkg.com/normalize.css/normalize.css">
+    <link rel="stylesheet" href="https://unpkg.com/milligram/dist/milligram.min.css">
+    <style>
+      body {
+        padding: 20px;
+        display: flex;
+        flex-direction: column;
+        min-height: 100vh;
+      }
+
+      div[role="main"] {
+        flex: 1;
+      }
+
+      footer {
+        text-align: center;
+      }
+
+      footer p {
+        margin-bottom: 0;
+      }
+
+      #twilio-logo {
+        width: 50px;
+        height: 50px;
+      }
+
+      #check-status {
+        padding: 1em;
+      }
+
+      .status-success {
+        color: darkgreen;
+      }
+
+      .status-failure {
+        color: maroon;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>Welcome to your new Twilio App</h1>
+    </header>
+    <div role="main">
+      <section>
+        <h3>Get started with your app</h3>
+        <p>This app automatically logs texts to your phone number into the specified Google Sheets document. Follow the steps below to test your app:</p>
+        <ol>
+          <li>Share your Google Sheets document with the email address of your Google Sheets service account.</li>
+          <li>Text your Twilio phone number with a message.</li>
+          <li>Your spreadsheet should now contain the SID, phone number, and content of the text you sent.</li>
+        </ol>
+      </section>
+      <section>
+        <h3>Troubleshooting</h3>
+        <ul>
+          <li>
+            Verify your Google Sheets configuration:
+            <div id="check-status">
+              <button onclick="showGoogleSheetsStatus()">Verify Configuration</button>
+              <span id="status-message"></span>
+            </div>
+            If the above check reveals problems, edit this function's <code>.env</code> file via the <code>Edit This Code</code> button below, if available, and ensure that all values are correct.
+          </li>
+          <li>Check the <a href="https://www.twilio.com/console/phone-numbers/incoming">phone number configuration</a> and make sure the Twilio phone number you want your app has an SMS webhook configured to point at
+            <p>
+              <pre><code><span class="function-root"></span>/log-sms</code></pre>
+            </p>
+          </li>
+          <li>Ensure that you have created a Google service account, and that <code>SHEETS_CLIENT_EMAIL</code> and <code>SHEETS_PRIVATE_KEY</code> contain the values from the <code>client_email</code> and <code>private_key</code> fields of its JSON key file, respectively.</li>
```

Comment:
Remember there won't be an `auth.json` for quick deployed apps, so i'm not sure what sort of info this would help provide

Reply:
Hmm, I probably need to rephrase this step a bit. The intent is for the user to verify that their `.env` contains the correct authentication information for their service account, since that's the most likely reason the app would fail to work. The trouble is, and I should clarify this, the Google Cloud console will never display this information in the browser; it's only available as a JSON file the user downloads. They'd then copy values from that file into their `.env`, rather than uploading it as an `auth.json`.